__tests__/app.test.js

```diff
@@ -271,3 +243,46 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Should return all articles in test db", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+
+        body.articles.forEach((articleObj) => {
+          expect(Object.keys(articleObj)).toHaveLength(8);
+          expect(articleObj).toEqual(expect.any(Object));
+        });
+
+        expect(body.articles).toHaveLength(12);
+      });
+  });
```

Comment:
These should be checked for contents using objectContaining rather than object.Keys to check the length as the keys could be different